### PR TITLE
Work

### PR DESCRIPTION
**Pull request by KevinLaCarrubba**
not gonna tell you about why i couldnt push from branch work to branch Work ... learning pains include capitals and colons

### CODE REVIEW

File: portfolio/src/components/Body/Contact/Contact.js [hunk 1]
```diff
@@ -4,22 +4,23 @@ import { GrMail } from "react-icons/gr";
 import contact from "./_contact.module.css";
 
 const Contact = () => {
+  const iconSize = 40;
   return (
     <div className={contact.contact}>
       {/* add icons for links */}
       <a href="#" target="_blank">
```
Comment (arfusop) [inline portfolio/src/components/Body/Contact/Contact.js:11]: You dont need a paragraph inside the anchor that holds an SVG element. Just do `<a><Icon/></a>`

File: portfolio/src/components/Body/Project/Project.js [hunk 2]
```diff
@@ -1,18 +1,14 @@
...
       </div>
-      <ul>
-        {/* make a component to view projects */}
-        <li>Project1</li>
-        <li>Project2</li>
-        <li>Project3</li>
-        <li>Project4</li>
-      </ul>
+      {/* make a component to view projects */}
+      <ProjectDisplay />
```
Comment (arfusop) [inline portfolio/src/components/Body/Project/Project.js:11]: Remove the `<> </>`, move the `ProjectDisplay` into the `div.projectContainer`. Use that div as the wrapper element for this entire component

File: portfolio/src/components/Body/Project/ProjectDisplay.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import nightout from "./Images/nightout.png";
+import project from "./_projectDisplay.module.css";
```
Comment (arfusop) [inline portfolio/src/components/Body/Project/ProjectDisplay.js:3]: Update the variable for your `project` styles to `styles`, common convention. If there's ever multiple css module files imported to one component, you'd want to name them as `projectStyles` and `[otherDescriptiveName]Styles`, but we'll cross that bridge when we get there

**Review by arfusop — CHANGES_REQUESTED** [review]

File: portfolio/src/components/Body/Contact/_contact.module.css [hunk 4]
```diff
@@ -1,9 +1,98 @@
...
-.contact p {
-  padding: 0.25rem;
+.title {
+  margin: 0;
+  color: white;
+  font-size: 4rem;
+  padding-bottom: 2rem;
+}
+
+/* .container {
```
Comment (arfusop) [inline portfolio/src/components/Body/Contact/_contact.module.css:17]: Get rid of commented out code. CSS you can very easily recreate, no need to clutter the code up 😆

File: portfolio/src/components/Body/Contact/Contact.js [hunk 5]
```diff
@@ -1,23 +1,29 @@
...
-        <FaLinkedinIn size={iconSize} />
-      </a>
-      <a href="#" target="_blank">
-        <FaGithub size={iconSize} />
-      </a>
-      <a href="#" target="_blank">
-        <GrMail size={iconSize} />
-      </a>
-    </div>
+    <section id="contact" className={styles.contact}>
```
Comment (arfusop) [inline portfolio/src/components/Body/Contact/Contact.js:8]: This ID is not needed

File: portfolio/src/components/Body/Contact/Contact.js [hunk 6]
```diff
@@ -1,23 +1,29 @@
...
+      <h1 className={styles.title}>Contact</h1>
+      <div className={styles.container}>
+        <a href="#" target="_blank" className={styles.icon1}>
```
Comment (arfusop) [inline portfolio/src/components/Body/Contact/Contact.js:11]: all of these `a` tags have a common theme, you want to apply very similar to the same styling in the children of all of them. Add an `id="icon[number]` to each, and give them all the same class: `iconLinks` (or whatever class name you want it to be. Then in the CSS, apply the `grid-area` by ID, and then apply the following styling
```
  text-decoration: none;
  display: flex;
  align-items: center;
  color: white;
  height: 100%;
  width: 100%;
```
to the general, reused class for all of the anchor tags

File: portfolio/src/components/Body/Contact/_contact.module.css [hunk 7]
```diff
@@ -1,9 +1,98 @@
 .contact {
```
Comment (arfusop) [inline portfolio/src/components/Body/Contact/_contact.module.css:1]: Move all of the styles outside of `.contact` inside, nest appropriately

File: portfolio/src/components/Body/Contact/_contact.module.css [hunk 8]
```diff
@@ -1,9 +1,98 @@
...
+  grid-auto-flow: row;
+  justify-items: start;
+  align-items: center;
+  grid-template-areas:
+    "icon1 ."
+    "icon2 quote"
+    "icon3 .";
+}
+
+.icon1 {
```
Comment (arfusop) [inline portfolio/src/components/Body/Contact/_contact.module.css:49]: Do what i said in the above PR about adding IDs, combining the styles. Then move these all nested under the `.container`, so this should be three levels deep of nesting at this point

File: portfolio/src/components/Body/Contact/_contact.module.css [hunk 9]
```diff
@@ -1,9 +1,98 @@
...
+  grid-area: icon3;
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+  color: white;
+  height: 100%;
+  width: 100%;
+}
+
+.quote {
```
Comment (arfusop) [inline portfolio/src/components/Body/Contact/_contact.module.css:79]: nest inside of `.container` as well

File: portfolio/src/components/Body/Contact/_contact.module.css [hunk 10]
```diff
@@ -1,9 +1,98 @@
...
+  grid-area: quote;
+}
+
+/* .link {
```
Comment (arfusop) [inline portfolio/src/components/Body/Contact/_contact.module.css:83]: delete

File: portfolio/src/components/Body/Contact/_contact.module.css [hunk 11]
```diff
@@ -1,9 +1,98 @@
...
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+  color: white;
+  height: 100%;
+  width: 100%;
+} */
+.icons {
```
Comment (arfusop) [inline portfolio/src/components/Body/Contact/_contact.module.css:91]: nest

File: portfolio/src/components/Body/Contact/_contact.module.css [hunk 12]
```diff
@@ -1,9 +1,98 @@
...
+  color: white;
+  font-size: 2.5rem;
+}
+.iconText {
```
Comment (arfusop) [inline portfolio/src/components/Body/Contact/_contact.module.css:95]: nest

File: portfolio/src/components/Body/Navbar/_navbar.module.css [hunk 13]
```diff
@@ -0,0 +1,67 @@
+.navbar {
```
Comment (arfusop) [inline portfolio/src/components/Body/Navbar/_navbar.module.css:1]: nest everything properly

File: portfolio/src/components/Body/Project/_project.module.css [hunk 14]
```diff
@@ -0,0 +1,18 @@
...
+  height: 100vh;
+  background: black;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.title {
```
Comment (arfusop) [inline portfolio/src/components/Body/Project/_project.module.css:11]: nest

File: portfolio/src/components/Body/Project/_projectDisplay.module.css [hunk 15]
```diff
@@ -21,10 +21,14 @@
...
+  background-color: #222222;
 }
 
 .siteDisplay:hover .overlay {
   width: 50%;
-  opacity: 0.69;
+  opacity: 0.9;
+}
+
+.imageTitle {
```
Comment (arfusop) [inline portfolio/src/components/Body/Project/_projectDisplay.module.css:32]: nest

File: portfolio/src/components/Footer/Footer.js [hunk 16]
```diff
@@ -1,11 +1,7 @@
 import React from "react";
 
 const Footer = () => {
-  return (
-    <div>
-      <footer className="footer">PARfuso Taught Dat Footer</footer>
-    </div>
-  );
+  return <div></div>;
```
Comment (arfusop) [inline portfolio/src/components/Footer/Footer.js:4]: If you're not using the footer at all, delete. If you plan to add a footer, ignore this for now

File: portfolio/src/components/Header/_header.module.css [hunk 17]
```diff
@@ -1,23 +1,59 @@
 .header {
```
Comment (arfusop) [inline portfolio/src/components/Header/_header.module.css:1]: nestttttt

**Review by arfusop — CHANGES_REQUESTED** [review]

File: portfolio/src/App.js [hunk 18]
```diff
@@ -1,15 +1,13 @@
...
-import Footer from "./components/Footer/Footer";
 import "./App.css";
 
 const App = () => {
   return (
     <main>
       <Header />
       <Body />
-      <Footer />
     </main>
```
[the remark this answers is not in the document]
Comment (KevinLaCarrubba) [reply portfolio/src/App.js:11]: footers fucked

File: portfolio/src/components/Body/AboutMe/_aboutme.module.css [hunk 19]
```diff
@@ -1,4 +1,18 @@
...
-  justify-content: center;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  border-bottom: 3px solid #222222;
+}
+
+.title {
+  font-size: 4rem;
+  color: rgba(24, 170, 151, 1);
```
[the remark this answers is not in the document]
Comment (KevinLaCarrubba) [reply portfolio/src/components/Body/AboutMe/_aboutme.module.css:14]: all colored up

File: portfolio/src/components/Body/Navbar/_navbar.module.css (hunk 13 above)
Comment (KevinLaCarrubba) [reply portfolio/src/components/Body/Navbar/_navbar.module.css:1]: take it ease with the nest

File: portfolio/src/components/Body/Project/_project.module.css (hunk 14 above)
Comment (KevinLaCarrubba) [reply portfolio/src/components/Body/Project/_project.module.css:11]: i said take it ease

File: portfolio/src/components/Footer/Footer.js (hunk 16 above)
Comment (KevinLaCarrubba) [reply portfolio/src/components/Footer/Footer.js:4]: footer be gone

File: portfolio/src/components/Header/_header.module.css (hunk 17 above)
Comment (KevinLaCarrubba) [reply portfolio/src/components/Header/_header.module.css:1]: I SAID TAKE IT EASY .. hold my grey !

File: portfolio/src/components/Body/AboutMe/AboutMe.js [hunk 20]
```diff
@@ -1,11 +1,10 @@
 import React from "react";
-import aboutme from "./_aboutme.module.css";
+import styles from "./_aboutme.module.css";
 const AboutMe = () => {
   return (
-    <div>
-      <h1 className={aboutme.aboutMe}>About Me</h1>
-      <p className={aboutme.aboutMe}>Yer Im Kevin Add info here.</p>
-    </div>
+    <section id="profile" className={styles.profile}>
```
[the remark this answers is not in the document]
Comment (KevinLaCarrubba) [reply portfolio/src/components/Body/AboutMe/AboutMe.js:5]: IDs have left the building .. make way for smooth scrolling

File: portfolio/src/components/Body/Contact/_contact.module.css (hunk 4 above)
Comment (KevinLaCarrubba) [reply portfolio/src/components/Body/Contact/_contact.module.css:17]: Win Guardian Live E OH Sah . its gone

File: portfolio/src/components/Body/Contact/Contact.js (hunk 5 above)
Comment (KevinLaCarrubba) [reply portfolio/src/components/Body/Contact/Contact.js:8]: poof

File: portfolio/src/components/Body/Contact/Contact.js (hunk 6 above)
Comment (KevinLaCarrubba) [reply portfolio/src/components/Body/Contact/Contact.js:11]: didnt think to use ID

File: portfolio/src/components/Body/Contact/_contact.module.css (hunk 12 above)
Comment (KevinLaCarrubba) [reply portfolio/src/components/Body/Contact/_contact.module.css:95]: you and the nest

**Review by arfusop — APPROVED** [review]

**Pull request merged**